content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -154,7 +154,9 @@ external IP address.
    The response to a successful request is a hello message:
 
    ```shell
-   Hello Kubernetes!
+   Hello, world!
+   Version: 2.0.0
+   Hostname: hello-world-cdd4458f4-m47c8
```

Comment:
_Very minor nit:_ In the output for "Hostname," it would be expected to see the request being routed to one of the Pods listed in the earlier steps. Currently, the Pod mentioned (" hello-world-cdd4458f4-m47c8") does not match any from the list. This might be the case for other tutorial output as well.

```suggestion
   Hostname: hello-world-2895499144-2e5uh
```

Reply:
Thanks for catching that, @dipesh-rawat. You're absolutely right about the hostname not matching the listed pods. I'll make the necessary correction. Appreciate the keen eye!